### PR TITLE
Bump router to haprox18

### PR DESCRIPTION
**Pull request by sdodson**

### CODE REVIEW

File: images/router/haproxy/Dockerfile [hunk 1]
```diff
@@ -5,7 +5,9 @@
 #
 FROM openshift/origin
 
-RUN INSTALL_PKGS="haproxy" && \
+RUN INSTALL_PKGS="haproxy18" && \
+    yum install yum-config-manager && \
+    yum-config-manager --add http://cbs.centos.org/repos/paas7-openshift-origin38-testing/x86_64/os/ && \
```
Comment (smarterclayton) [inline images/router/haproxy/Dockerfile]: Why is the haproxy18 package in the normal set of dependencies we already set?  We spent 2 releases trying to stop doing it.

Comment (eparis) [reply images/router/haproxy/Dockerfile]: I assume you meant "why ISN'T the haproxy18..."

**Comment by smarterclayton** [comment]
What is the resolution of the concerns raised earlier?  This is going to be fixed tomorrow?

**Comment by eparis** [comment]
We hope tomorrow to revert the repo swizzle. Right now we are just trying to get it to work at all...

**Comment by smarterclayton** [comment]
Ok, that's fine.

**Pull request merged**